src/api/app/assets/stylesheets/webui/user.scss

```
@@ -1,38 +1,30 @@
-.activity-percentil0 {
-  background-color: #dcd5da;
-}
-
-.activity-percentil1 {
-  background-color: #f9b4dd;
-}
-
-.activity-percentil2 {
-  background-color: #ff5cb3;
-}
-
-.activity-percentil3 {
-  background-color: #ec1596;
-}
-
-.activity-percentil4 {
-  background-color: #c60b7b;
-}
-
 #contributors-table {
   td {
     padding: 0;
-    border: 1px solid white;
+    border: 1px solid $gray-400;
   }
   td.week-day {
     width: 1px;
-    padding-right: 1rem
+    padding: 0 .5rem 0 .5rem;
+    @extend .table-activity;
   }
   a.activity-link {
     display: block;
-    text-decoration: none;
   }
 }
 
 #bio-chars-counter {
   min-height: 1.2rem;
 }
+
+$table-activity-variants: (
+  "activity":            shift-color($gray-400, -70%),
+  "activity-percentil1": shift-color($pink-300, -70%),
+  "activity-percentil2": shift-color($pink-500, -70%),
+  "activity-percentil3": shift-color($pink-700, -70%),
+  "activity-percentil4": shift-color($pink-900, -70%),
```

```suggestion
  "activity":            shift-color($gray-400, -50%),
  "activity-percentil1": shift-color($pink-300, -50%),
  "activity-percentil2": shift-color($pink-500, -50%),
  "activity-percentil3": shift-color($pink-700, -50%),
  "activity-percentil4": shift-color($pink-900, -50%),
```

Just a suggestion, it feels a little bit too bright and transparent with the `-70%` to me, what do you think?

---

The default uses `-80%`. See: https://github.com/twbs/bootstrap/blob/v5.3.5/scss/_variables.scss#L767-L781

I changed it to `-70%` because of the same reason you pointed out.

I'm find with changing it to '-50%'. Changed. :+1:

---

I think the default it's because it is meant to be a table cell background with maybe some text on top, here we're just painting the cell so I think it is fine to customize it a little bit. Cool!! :+1: